### DISCUSSION TITLE
DISABLE DHCP

### DISCUSSION THREAD

**Discussion by PincoPanco** [dchristl/esp32_nat_router_extended#128] [General]
HI
In order to increase the wifi area  of my video surveillance system I have installed two ESP32 extender in my garden. Both have unique IP assigned by DHCP server via a stsic DHCP table. Both extenders assign to the connected video camera the IP address 192.168.4.2 ( my network class is 192.168.10.xx) so I can't address individually the video camera. 
Why the dhcp server assign to the connected devices IP address with network class different from the one assigned?
The extender DHCP server can be disabled?
Thanks for your attention
marco

**Comment by dchristl** [comment]
Hi,

I honestly don't understand what your problem is. You have 2 ESP32 connected to the same router (static IP). So far so good, from now on I don't understand anything. The ESP have a completely new network and of course need a DHCP. The IP addresses for your camera also come from the ESP and not your router. This is not a mesh or similar. These are two completely different networks. I think for what you want to achieve, you need to deactivate NAT and define a static route, like [here](https://community.home-assistant.io/t/declare-a-static-route-to-reach-other-lan/311442) ? This was also discussed in #73 

Kind regards,
Danny
